kafka11/src/main/java/zipkin2/reporter/kafka11/KafkaSender.java

```diff
@@ -127,6 +134,29 @@ public final Builder overrides(Map<String, ?> overrides) {
       return this;
     }
 
+    /**
+     * By default, a producer will be created, targeted to {@link #bootstrapServers(String)} with 0
+     * required {@link ProducerConfig#ACKS_CONFIG acks}. Any properties set here will affect the
+     * producer config.
+     *
+     * Consider not overriding batching properties ("batch.size" and "linger.ms") as those will
+     * duplicate buffering effort that is already handled by Sender.
+     *
+     * <p>For example: Reduce the timeout blocking from one minute to 5 seconds.
+     * <pre>{@code
+     * Properties overrides = new Properties();
+     * overrides.put(ProducerConfig.MAX_BLOCK_MS_CONFIG, 5000);
+     * builder.overrides(overrides);
+     * }</pre>
+     *
+     * @see ProducerConfig
+     */
+    public final Builder overrides(Properties overrides) {
```

Comment:
weird.. i thought properties would have matched on signature map string ?

Reply:
Yeap, it works with `Map<?,?>` as Properties implemets `Hashtable<Object, Object>` but that's a bit weirder IMHO. KafkaProducer has a `propsToMap`: https://github.com/apache/kafka/blob/201da0542726472d954080d54bc585b111aaf86f/clients/src/main/java/org/apache/kafka/clients/producer/KafkaProducer.java#L1196